extra-files/cloud_setup/shed_tool_conf.xml.cloud

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
```

Comment:
I like this, but if galaxy was able to use the tool_panel_section label to place tools in pre-existing, we wouldn't need this, right? I think that would be a nice improvement that isn't too hard to do.